comp/core/tagger/params.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package tagger
+
+// AgentTypeForTagger represents agent types that tagger is used for
+type AgentTypeForTagger uint8
+
+// Define agent type for tagger
+const (
+	LocalTaggerAgent AgentTypeForTagger = 1 << iota
+	NodeRemoteTaggerAgent
+	CLCRunnerRemoteTaggerAgent
+	FakeTagger
+)
+
+// Params provides the kind of agent we're instantiating workloadmeta for
+type Params struct {
+	AgentTypeForTagger AgentTypeForTagger
```

Comment:
What do you think about making `AgentTypeForTagger` unexported and having named constructors for each type? It should reduce typing for the caller.